templates/direct_menu.php

```diff
@@ -26,9 +26,13 @@
 
 	#navigation div ul {
 		position: relative;
-		margin-left: 170px;
+		padding-right:10px;
```

Comment:
the padding right on the <ul> element is not affecting the space between the icons, it is just the space after the last icon, which has no influence on the vanilla nextcloud theme. :wink: See comments on #54 on how you can integrate this withing your theme.